src/cprover/Makefile

```diff
@@ -0,0 +1,65 @@
+SRC = address_taken.cpp \
```

Comment:
Random place for comment: please also add to CMake infrastructure.

Reply:
done